internal/storage/storageutil/object_attrs_test.go

```diff
@@ -0,0 +1,102 @@
+package storageutil
+
+import (
+	"crypto/md5"
+	"testing"
+	"time"
+
+	"cloud.google.com/go/storage"
+	. "github.com/jacobsa/ogletest"
+	storagev1 "google.golang.org/api/storage/v1"
+)
+
+const TestBucketName string = "gcsfuse-default-bucket"
+const TestObjectName string = "gcsfuse/default.txt"
+const Test string = "test"
+
+func TestObjectAttrsToBucketObject(t *testing.T) { RunTests(t) }
+
+type objectAttrsTest struct {
+}
+
+func init() { RegisterTestSuite(&objectAttrsTest{}) }
+
+func (t objectAttrsTest) TestObjectAttrsToBucketObjectMethod() {
+	var attrMd5 []byte
+	Time := time.Now()
```

Comment:
variable names should start with small letter

Reply:
fixed